tokens.xml

```diff
@@ -9837,6 +10019,62 @@ Whenever a card is put into your graveyard from anywhere, you may return it to y
             <token>1</token>
             <tablerow>1</tablerow>
         </card>
+        <card>
+            <name>Day</name>
+            <prop>
+                <type>State</type>
+                <maintype>State</maintype>
+            </prop>
+            <set picURL="https://c1.scryfall.com/file/scryfall-cards/large/front/9/c/9c0f7843-4cbb-4d0f-8887-ec823a9238da.jpg?1630606483">MID</set>
+            <related attach="attach">Night<related>
+            <reverse-related exclude="exclude"></reverse-related>
```

Comment:
why are these empty?

Reply:
Leftover from when I was indecisive on whether to have Day related to Night or Night reverse-related to Day.